source/kernel/src/forth/mod.rs

```diff
@@ -615,7 +611,12 @@ impl BagOfHolding {
             if self.idx == 0 {
                 continue;
             }
-            if !self.inner.iter().any(|(idx, _)| *idx == self.idx) {
+            if !self
+                .inner
+                .as_slice()
```

Comment:
nit, TIOLI: it seems like this could be a `Deref` impl?

Reply:
I think there was a reason I didn't do that, maybe to leave us open to deref/derefmut something else? I'll think about while I'm doc'ing.